diff --git a/bin/reinstall_chrome_beta_mac.sh b/bin/reinstall_chrome_beta_mac.sh
index b6e1990cfc8b3f3cae717dd19f5b4a9470d12b6f..a8211e157273e33e1c26dfe1cff9c79c12575d2a 100755
--- a/bin/reinstall_chrome_beta_mac.sh
+++ b/bin/reinstall_chrome_beta_mac.sh
@@ -4,7 +4,7 @@ set -x
 
 rm -rf "/Applications/Google Chrome Beta.app"
 cd /tmp
-curl -o ./googlechromebeta.dmg -k https://dl.google.com/chrome/mac/universal/beta/googlechromebeta.dmg
+curl -o ./googlechromebeta.dmg https://dl.google.com/chrome/mac/universal/beta/googlechromebeta.dmg
 hdiutil attach -nobrowse -quiet -noautofsck -noautoopen -mountpoint /Volumes/googlechromebeta.dmg ./googlechromebeta.dmg
 cp -pR "/Volumes/googlechromebeta.dmg/Google Chrome Beta.app" /Applications
 hdiutil detach /Volumes/googlechromebeta.dmg
diff --git a/bin/reinstall_chrome_stable_mac.sh b/bin/reinstall_chrome_stable_mac.sh
index 91d826c0148e1fb0396cf787371f97342903e98c..b5ab4172dc64c20049ad58b6b9b1c5ce42266f0e 100755
--- a/bin/reinstall_chrome_stable_mac.sh
+++ b/bin/reinstall_chrome_stable_mac.sh
@@ -4,7 +4,7 @@ set -x
 
 rm -rf "/Applications/Google Chrome.app"
 cd /tmp
-curl -o ./googlechrome.dmg -k https://dl.google.com/chrome/mac/universal/stable/GGRO/googlechrome.dmg
+curl -o ./googlechrome.dmg https://dl.google.com/chrome/mac/universal/stable/GGRO/googlechrome.dmg
 hdiutil attach -nobrowse -quiet -noautofsck -noautoopen -mountpoint /Volumes/googlechrome.dmg ./googlechrome.dmg
 cp -pR "/Volumes/googlechrome.dmg/Google Chrome.app" /Applications
 hdiutil detach /Volumes/googlechrome.dmg
diff --git a/bin/reinstall_msedge_beta_mac.sh b/bin/reinstall_msedge_beta_mac.sh
index 69c0602485afd1015089a53fe5ddca0176d8e6d2..6e411480018b4b59deb4459d451cdda54ecc70e9 100755
--- a/bin/reinstall_msedge_beta_mac.sh
+++ b/bin/reinstall_msedge_beta_mac.sh
@@ -3,7 +3,7 @@ set -e
 set -x
 
 cd /tmp
-curl -o ./msedge_beta.pkg -k "$1"
+curl -o ./msedge_beta.pkg "$1"
 # Note: there's no way to uninstall previously installed MSEdge.
 # However, running PKG again seems to update installation.
 sudo installer -pkg /tmp/msedge_beta.pkg -target /
diff --git a/bin/reinstall_msedge_dev_mac.sh b/bin/reinstall_msedge_dev_mac.sh
index 0ad05b0a2b2e9a53d3a45f327324bea03b1a463f..84c7d474cf58f669c7a3368afe3ac741423bad5d 100755
--- a/bin/reinstall_msedge_dev_mac.sh
+++ b/bin/reinstall_msedge_dev_mac.sh
@@ -3,7 +3,7 @@ set -e
 set -x
 
 cd /tmp
-curl -o ./msedge_dev.pkg -k "$1"
+curl -o ./msedge_dev.pkg "$1"
 # Note: there's no way to uninstall previously installed MSEdge.
 # However, running PKG again seems to update installation.
 sudo installer -pkg /tmp/msedge_dev.pkg -target /
diff --git a/bin/reinstall_msedge_stable_mac.sh b/bin/reinstall_msedge_stable_mac.sh
index b82cfb37cd20364108825760902538cb461fa379..9532cbcd1b1680c11ffc1f34d44173ccfed8a27d 100755
--- a/bin/reinstall_msedge_stable_mac.sh
+++ b/bin/reinstall_msedge_stable_mac.sh
@@ -3,7 +3,7 @@ set -e
 set -x
 
 cd /tmp
-curl -o ./msedge_stable.pkg -k "$1"
+curl -o ./msedge_stable.pkg "$1"
 # Note: there's no way to uninstall previously installed MSEdge.
 # However, running PKG again seems to update installation.
 sudo installer -pkg /tmp/msedge_stable.pkg -target /
